lib/shopify-cli/commands/generate.rb

```diff
@@ -66,8 +56,8 @@ def self.extended_help
         HELP
       end
 
-      def self.run_generate(script, name, ctx)
-        stat = ctx.system(script)
+      def self.run_generate(script, name)
+        stat = @ctx.system(script)
```

Comment:
I don't think this will work because its a class method, which is why it was passed in

Reply:
I know you're not wrong, but it does work. Why would it work? It totally runs the script to generate all the commands.

Reply:
I am worried about this. If we don't know how it works, we should probably not use it. Why did you change it?

Reply:
Hey! I changed it back, but I do really want to know why it works.